doc/negate.txt

```diff
@@ -0,0 +1,11 @@
+* _.negate(f)
```

Comment:
While negation does apply to boolean algebra, one does risk confusing it with unary minus. I'd say complement, but that sounds too much like sets. Maybe `inverted`?

Reply:
https://lodash.com/docs/4.17.10#negate